ext/dav1d.cmd

```diff
@@ -3,22 +3,17 @@
 
 : # The odd choice of comment style in this file is to try to share this script between *nix and win32.
 
-: # meson and ninja must be in your PATH.
+: # meson must be in your PATH.
```

Comment:
Undo this change? Meson requires Ninja:
https://mesonbuild.com/Getting-meson.html#dependencies